### DISCUSSION TITLE
Çeviri Botu Entegresi

### DISCUSSION THREAD

**Discussion by Majezikonline** [IBAX-io/go-ibax#452] [Ideas] [answered]
### Clear and concise description of the problem

İbax formuna özel "dil çevirici" botun site içerisine entegre edilip, her dilden kullanıcının rahat bir şekilde site içerisinde gezinebilmesi.

### Suggested solution

Dil çevirme botunun siteye entegre edilmesi ve otomatik olarak dil çevirmesi.

### Alternative

Dillere göre moderatör alımları

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by ibaxio** [comment, marked as answer]
The IBAX team is recruiting volunteers to translate the official website. Due to the rapid growth of the Turkish community, Turkish will be uploaded in the next revision.

We also welcome translations from community members in other languages.

Unleash the power of our community and make us stronger.
